src/renderer/components/WebPlatformPlayer/LiveAppDisclaimer.js

```diff
@@ -0,0 +1,142 @@
+import { rgba } from "~/renderer/styles/helpers";
+import Image from "~/renderer/components/Image";
+import Text from "~/renderer/components/Text";
+import { openURL } from "~/renderer/linking";
+import Box from "~/renderer/components/Box";
+
+type Props = {
+  manifest: AppManifest,
+};
+
+const Head: ThemedComponent<{}> = styled(Box).attrs(p => ({
+  horizontal: true,
+  alignItems: "center",
+  justifyContent: "center",
+  mb: 24,
+}))`
+  > * {
+    margin: 0 4px;
+  }
+`;
+
+const Dashes: ThemedComponent<{}> = styled.div`
+  width: 40px;
+  opacity: 0.5;
+  height: 3px;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg'%3E%3Cline x1='1' y1='1.5' x2='39' y2='1.5' stroke='%23${p =>
+    p.theme.colors.palette.primary.main.slice(
+      1,
+    )}' stroke-opacity='0.5' stroke-width='2' stroke-linecap='round' stroke-dasharray='2 6'/%3E%3C/svg%3E%0A");
+`;
+
+const LogoContainer: ThemedComponent<{}> = styled(LedgerLiveLogo)`
+  box-shadow: initial;
+  height: 48px;
+  width: 48px;
+  border: 1px solid #ececec;
+`;
+
+const Title: ThemedComponent<{}> = styled(Text).attrs(p => ({
+  mb: 12,
+  fontSize: 22,
+  textAlign: "center",
+  ff: "Inter|SemiBold",
+}))``;
+
+const Description: ThemedComponent<{}> = styled(Text).attrs(p => ({
+  color: rgba(p.theme.colors.dark, 0.6),
+  ff: "Inter|Regular",
+  mb: 12,
+  textAlign: "center",
+  fontSize: 14,
+}))``;
+
+const ExternalLinkContainer: ThemedComponent<{}> = styled(Text).attrs(p => ({
+  color: p.theme.colors.palette.primary.main,
+  ff: "Inter|Medium",
+  fontSize: 13,
+  textAlign: "center",
+  mb: 12,
+}))`
+  > div {
+    text-decoration: none;
+  }
+`;
+
+const BlueInfoContainer: ThemedComponent<{}> = styled(Box).attrs(p => ({
+  flex: 1,
+  p: 12,
+  horizontal: true,
+  alignItems: "center",
+  mt: 28,
+}))`
+  background-color: ${p => rgba(p.theme.colors.palette.primary.main, 0.1)};
+`;
+
+const InfoIconContainer: ThemedComponent<{}> = styled(Box).attrs(p => ({
+  mr: "6px",
+}))``;
+
+const InfoText: ThemedComponent<{}> = styled(Text).attrs(p => ({
+  ml: "6px",
+  ff: "Inter|Medium",
+  fontSize: 12,
+  lineHeight: 18,
+  color: p.theme.colors.palette.primary.main,
+}))`
+  flex: 1;
+  line-height: 18px;
+  white-space: pre-wrap;
+`;
+
+export const LiveAppDisclaimer = ({ manifest }: Props) => {
+  const { colors } = useTheme();
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Head>
+        <LogoContainer
+          icon={<Image resource={LedgerLiveImg} alt="" draggable="false" width={31} height={31} />}
+        />
+        <Dashes />
+        <LiveAppIcon size={48} name={manifest.name} icon={manifest.icon || ""} />
+      </Head>
+
+      <Title>{t("platform.disclaimer.title")}</Title>
+
+      <Description>{t("platform.disclaimer.description")}</Description>
+
+      <ExternalLinkContainer>
+        <ExternalLink
+          label="Learn more"
+          isInternal={false}
+          onClick={() => openURL(manifest.homepageUrl)}
```

Comment:
General question: is it normal that this learn more link redirect to the actual app website? It seems unintuitive, I would wait to be redirect to a ledger page to actually learn more about this "You are about to be redirected to an application not operated by Ledger" informative message 🤔 

PS: [The Figma link](https://www.figma.com/file/YKPMz1tPJC70zDQVtUl4Ot/Discover-LLD?node-id=2699%3A34532) provided in Jira seems broken (it does not redirect to the corresponding screen) so I can't check / post a comment on it

Reply:
You're right, I was still thinking about the Info Panel and thought it was the same kind of external link. I actually have no idea where it's supposed to go if it's not to the homepage url :/

Reply:
Link has been removed from the Figma by Chris (since it's not on mobile at all), I'm removing it and rebasing 👍

Reply:
Done!